src/internal/components/file-input/interfaces.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { BaseComponentProps } from '../../base-component';
+import { FormFieldCommonValidationControlProps } from '../../context/form-field-context';
+import { NonCancelableEventHandler } from '../../events';
+
+export interface FileInputProps extends BaseComponentProps, FormFieldCommonValidationControlProps {
+  /**
+   * Variant of the file input. Defaults to "button".
+   */
+  variant?: 'button' | 'icon';
+
+  /**
+   * Adds `aria-label` to the file input element. Use this to provide an accessible name for file inputs
+   * that don't have visible text, and to distinguish between multiple file inputs with identical visible text.
+   */
+  ariaLabel?: string;
+
+  /**
+   * Text of label (default property) - used as aria label for button variant.
```

Comment:
What's default property?

Reply:
It's the property that shows up as children... ie
```
<FileInput>{children}</FileInput
```

I am copying the [button component's](https://github.com/cloudscape-design/components/blob/main/src/button/interfaces.ts#L83) text property here.

But good call, we don't say "(default property)" specifically there, so I'll remove from the description here.